packages/theme/src/cli/commands/theme/pull.ts

```diff
@@ -53,6 +55,12 @@ export default class Pull extends ThemeCommand {
       description: 'Proceed without confirmation, if current directory does not seem to be theme directory.',
       env: 'SHOPIFY_FLAG_FORCE',
     }),
+    stable: Flags.boolean({
+      hidden: true,
+      description:
+        'Performs the upload by relying in the legacy download approach (slower, but it might be more stable in some scenarios)',
```

Comment:
Just noted some copypasta 🍝 

```suggestion
        'Performs the download by relying on the legacy Ruby code',
```

Reply:
Good catch!